src/components/apis/details-of-api/detailsOfApiModel.ts

```diff
@@ -1,8 +1,14 @@
 import { HyperlinkModel } from "@paperbits/common/permalinks";
+import { LocalStyles } from "@paperbits/common/styles";
 
-export class DetailsOfApiModel {    
+export class DetailsOfApiModel {
     /**
      * Link to a page that contains API changelog.
      */
     public changeLogPageHyperlink: HyperlinkModel;
+
+    /**
+     * Widget local styles.
+     */
+    styles: LocalStyles;
```

Comment:
This is a class so if you require that field always need to have object under that field you can do the following

```suggestion
    styles: LocalStyles = {};
```